--- exercise-kdelibsobs-test.solution	2005-11-05 21:30:38.000000000 +0100
+++ exercise-kdelibsobs-test.mistake	2005-11-30 10:37:36.000000000 +0100
@@ -1,9 +0,0 @@
->!> Installing kdelibs3-devel from channel test
->!> Installing kdelibs3 from channel test
->!> Solution #1:
->!> install kdelibs3-3.1-73[test]
->!> install kdelibs3-devel-3.1-73[test]
->!> remove  kdelibs-devel-2.0.1-4
->!> remove  kdesupport-2.0.1-5
->!> remove  kdesupport-devel-2.0.1-5
->!> installs=2, upgrades=0, uninstalls=3
