--- exercise-bug21829-5-test.solution	2005-11-05 21:10:50.000000000 +0100
+++ exercise-bug21829-5-test.mistake	2005-11-30 10:37:36.000000000 +0100
@@ -1,11 +0,0 @@
->!> Installing lsb from channel test
->!> Solution #1:
->!> install lprng-1.0-1[test]
->!> install lsb-1.0-1[test]
->!> |unflag cups-client-1.0-1[test]
->!> installs=2, upgrades=0, uninstalls=0
->!> Solution #2:
->!> install cups-client-1.0-1[test]
->!> install lsb-1.0-1[test]
->!> |unflag lprng-1.0-1[test]
->!> installs=2, upgrades=0, uninstalls=0
