--- exercise-02conflict-09-test.solution	2005-11-04 23:15:23.000000000 +0100
+++ exercise-02conflict-09-test.mistake	2005-11-30 10:37:37.000000000 +0100
@@ -1,10 +0,0 @@
->!> Installing baz2 from channel test
->!> Solution #1:
->!> install baz2-1.0-1[test]
->!> remove  foo2-1.0-1
->!> installs=1, upgrades=0, uninstalls=1
->!> Solution #2:
->!> install baz2-1.0-1[test]
->!> upgrade foo2-1.0-1 => foo2-2.0-1[test]
->!> installs=1, upgrades=1, uninstalls=0
-
