--- exercise-05reallive-2-test.solution	2005-11-06 11:18:22.000000000 +0100
+++ exercise-05reallive-2-test.mistake	2005-11-30 10:37:37.000000000 +0100
@@ -1,11 +0,0 @@
->!> Installing inn from channel test
->!> Installing isdn4k-utils from channel test
->!> Installing kdbg from channel test
->!> Solution #1:
->!> install XFree86-libs-3.3.5-1[test]
->!> install gdbm-1.8.0-3[test]
->!> install inn-2.2.1-2[test]
->!> install isdn4k-utils-3.0beta2-4[test]
->!> install kdbg-1.0beta1-1[test]
->!> install kdelibs-1.1.2-1[test]
->!> installs=6, upgrades=0, uninstalls=0
