--- exercise-02conflict-06-test.solution	2005-11-05 17:03:31.000000000 +0100
+++ exercise-02conflict-06-test.mistake	2005-11-30 10:37:37.000000000 +0100
@@ -1,10 +0,0 @@
->!> Installing test3 from channel test
->!> Solution #1:
->!> install test3-1.0-1[test]
->!> remove  test1-1.0-1
->!> remove  test2-1.0-1
->!> |unflag test4-1.0-1[test]
->!> installs=1, upgrades=0, uninstalls=2
->!> Solution #2:
->!> This solution is a duplicate.
->!> installs=1, upgrades=0, uninstalls=2
