--- bug217574-test.solution	2006-12-20 15:01:03.000000000 +0100
+++ bug217574-test.mistake	2006-12-20 15:23:58.000000000 +0100
@@ -25,7 +25,6 @@
 >!> remove  nfs-utils-1.0.7-36.10.i586
 >!> remove  nfsidmap-0.12-16.2.i586
 >!> remove  ntfsprogs-1.11.2-15.2.i586
->!> remove  pptp-1.7.0-13.2.i586
 >!> remove  screen-4.0.2-62.2.i586
 >!> remove  selection:default-10.0-115.noarch
 >!> remove  smpppd-1.59-19.2.i586
@@ -37,4 +36,4 @@
 >!> remove  ypbind-1.19.1-14.4.i586
 >!> remove  zip-2.31-15.2.i586
 >!> remove  zsh-4.2.6-15.2.i586
->!> installs=0, upgrades=0, uninstalls=37
+>!> installs=0, upgrades=0, uninstalls=36
